store/sqlstore/upgrade.go

```diff
@@ -849,6 +849,8 @@ func upgradeDatabaseToVersion528(sqlStore SqlStore) {
 		sqlStore.AlterColumnTypeIfExists("IncomingWebhooks", "Username", "varchar(255)", "varchar(255)")
 		sqlStore.AlterColumnTypeIfExists("IncomingWebhooks", "IconURL", "text", "varchar(1024)")
 
+		sqlStore.AlterColumnTypeIfExists("Threads", "Who", "longtext", "text")
```

Comment:
This should happen in a new migration, not the 27 to 28 one. And, probably we don't need it if we define it properly in the sqlstore.